ble/DiscoveredCharacteristic.h

```diff
@@ -52,19 +56,88 @@ class DiscoveredCharacteristic {
         uint8_t _authSignedWrite :1; /**< Writing the value with Signed Write Command permitted. */
 
     public:
-        bool broadcast(void)       const {return _broadcast;      }
-        bool read(void)            const {return _read;           }
-        bool writeWoResp(void)     const {return _writeWoResp;    }
-        bool write(void)           const {return _write;          }
-        bool notify(void)          const {return _notify;         }
-        bool indicate(void)        const {return _indicate;       }
-        bool authSignedWrite(void) const {return _authSignedWrite;}
+        /**
+         * @brief   Check the value of the Properties_t::_broadcast property
```

Comment:
I will change it, but I do not think we are leaking implementation details because Properties_t is public anyways.